### ISSUE TITLE
Ordered list applied to multiple lines only adds 1. to each line

### ISSUE THREAD

**Issue by tonyhogben** [sparksuite/simplemde-markdown-editor#563]
When you select multiple lines and click the order list toolbar option, 1. is add to all lines rather than incremental numbers.

**Comment by jhpratt** [comment]
That's standard markdown. The interpreter renders it as consecutive numbers.

**Comment by tonyhogben** [comment]
I agree that it is acceptable markdown markup, however I would argue that the editor should display incremental numbers to not confuse less technical users. 

I would add that I do not know of any other markdown editors that do not display incremental numbers in the editor.

**Comment by dnajd** [comment]
This is definitely a bug.  

If you start an ordered list and add items it numbers them correctly...
```
   1. test
   2. test second
   3. test third
```
but if you write three items...
```
   test
   test second
   test third
```
highlight and turn into an ordered list you get

```
   1. test
   1. test second
   1. test third
```

It rendered as all '1.' in edit mode, but previewing fixes it.. and going back to edit mode from preview will also fix the glitch.  But this is confusing to users.

**Comment by deepsan98** [comment]
Are there any updates regarding this issue?
